hostprocess/build.sh

```diff
@@ -1,8 +1,22 @@
 #!/bin/bash
+<<'DESCRIPTION'
+  Script to build flannel and callico versions as well as their kube-proxy versions
+  There is also the posibility to build only a specific kube-proxy image or change the repository
+  and flannel/calico versions via the environment variables in the script,
+  in order to build it for other versions of them
+  
+  .EXAMPLE
+  ./build.sh --KubeProxyVersion v1.24.3
+  ./build.sh --KubeProxyVersion v1.24.3 -r sigwindowstools --flannelVersion v0.14.0
+  ./build.sh --KubeProxyVersion v1.24.3 -r sigwindowstools --calicoVersion v3.23.0
+
+DESCRIPTION
 
 repository=${repository:-"sigwindowstools"}
 flannelVersion=${flannelVersion:-"v0.14.0"}
-calicoVersion=${calicoVersion:-"v3.23.3"}
+calicoVersion=${calicoVersion:-"v3.23.0"}
+KubeProxyVersion=${KubeProxyVersion:-""}
```

Comment:
nit: this isn't lowercase like the other variables

Reply:
Changed it to `kubeProxyVersion`